### PR TITLE
doc entries on trimming the actionlogrecord table. (#5916)

### LINKED ISSUE #5916: Archive ActionLogRecord entries
In the current implementation, action log entries keep accumulating unimpeded. 
A quick (and somewhat anti-scientific) check suggests that our own prod. database is more than 50% action log entries by size, by now. 

(the test performed was comparing the total output of ``pg_dump`` of the full database, vs. ``pg_dump --table actionlogrecord``. It's "anti-scientific" because the output is generated as sql text that does not correspond 1:1 to the size of the data structures stored. But the ratio should be about right as a ballpark).

### PR DESCRIPTION
**Pull request by landreev**
**What this PR does / why we need it**:

Entries for the admin guide on how to keep the actionlogrecord table in trim.

**Which issue(s) this PR closes**:

Closes #5916

**Special notes for your reviewer**:

As I was looking for a place in the guide for this info, I realized once again that we don't have a doc section for low level sysadmin stuff (past/outside of the installation guide). 
But we may want to rename the troubleshooting section "troubleshooting, hacks and tips" at some point - ? 

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

**Additional documentation**:

### CODE REVIEW

File: doc/sphinx-guides/source/admin/troubleshooting.rst [hunk 1]
```diff
@@ -148,6 +148,27 @@ Many Files with a File Type of "Unknown", "Application", or "Binary"
 
 From the home page of a Dataverse installation you can get a count of files by file type by clicking "Files" and then scrolling down to "File Type". If you see a lot of files that are "Unknown", "Application", or "Binary" you can have the Dataverse  installation attempt to redetect the file type by using the :ref:`Redetect File Type <redetect-file-type>` API endpoint.
 
+.. _actionlogrecord-trimming:
+
+What's With this Table "ActionLogRecord" in Our Database, it Seems to be Growing Uncontrollably?
```
Comment (pdurbin) [inline doc/sphinx-guides/source/admin/troubleshooting.rst:153]:
```suggestion
What's with this Table "ActionLogRecord" in Our Database, It Seems to be Growing Uncontrollably?
```

File: doc/sphinx-guides/source/admin/troubleshooting.rst [hunk 2]
```diff
@@ -148,6 +148,27 @@ Many Files with a File Type of "Unknown", "Application", or "Binary"
...
...l for investigating problems or usage patterns. However, please note that there is no builtin mechanism in the Application for trimming this table, so it will keep continue growing as your Dataverse installation is kept in operation. For example, ultiple entries in this table are created every time a guest user views the page of a published dataset. Many more are created when an author is actively working on a dataset, making edits, adding new files, etc. On a busy installation this table is likely to grow at a faster rate than the actual data holdings. For example, after five years of production use at Harvard IQSS, the raw size of ActionLogRecord appeared to exceed the combined size of the rest of the database (!). It's worth pointing out that the sheer size of this one table does not by itself result in performance issues in any linear way. But it may still be undesirable to keep that much extra data around; especially since for most installations these records are unlikely to have much value past a certain number of months or years. Some installation may be purchasing their database services from cloud computing providers (RDS, etc.) where extra data may result in higher costs.
```
Comment (pdurbin) [inline doc/sphinx-guides/source/admin/troubleshooting.rst:156]: Typo: "ultiple"

**Review by pdurbin — COMMENTED** [review]
Looks good. Found a typo.

File: doc/sphinx-guides/source/admin/troubleshooting.rst (hunk 2 above)
Comment (scolapasta) [inline doc/sphinx-guides/source/admin/troubleshooting.rst:156]: Minor note, but rows are also created for some other "actions", e.g logging in, which are outside the commands. (that may be the only one?) Not sure if it's worth specifying that here or not.

**Review by scolapasta — COMMENTED** [review]
Looks good. I made one small comment and @pdurbin found a typo. Please review and then we can move it over.

**Review by scolapasta — APPROVED** [review]

**Pull request merged**